apps/tasks.py

```diff
@@ -636,3 +636,21 @@ def purge_tasks():
     Remove tasks from queue to avoid overflow
     """
     app.control.purge()
+
+
+@app.task
+def delete_old_objects():
+    '''
```

Comment:
Use `"`
Black would have fixed it, so don't forget pre-commit hooks :))

Reply:
Thanks! I've fixed it now!